internal/commands/tunnel.go

```diff
@@ -221,277 +157,145 @@ func (c *tunnelCmd) SSHKeyEnsurePresent() error {
 
-	ssmSvc := ssm.New(sess)
+	svc := ssm.New(sess)
 
-	out, err := ssmSvc.GetParameter(&ssm.GetParameterInput{
+	resp, err := svc.GetParameter(&ssm.GetParameterInput{
 		Name:           aws.String(fmt.Sprintf("/%s/terraform-output", "dev")),
 		WithDecryption: aws.Bool(true),
 	})
 	if err != nil {
-		pterm.Error.Printfln("Getting tunnel up command")
+		pterm.Error.Printfln("Getting SSH forward config")
 		return err
 	}
 
 	var value []byte
 
-	value, err = base64.StdEncoding.DecodeString(*out.Parameter.Value)
+	value, err = base64.StdEncoding.DecodeString(*resp.Parameter.Value)
 	if err != nil {
-		pterm.Error.Printfln("Getting tunnel up command")
+		pterm.Error.Printfln("Getting SSH forward config")
 		return err
 	}
 
-	var tOut terraformOutput
+	var config terraformOutput
 
-	err = json.Unmarshal(value, &tOut)
+	err = json.Unmarshal(value, &config)
 	if err != nil {
-		pterm.Error.Printfln("Getting tunnel up command")
+		pterm.Error.Printfln("Getting SSH forward config")
 		return err
 	}
 
-	command := strings.Split(tOut.Cmd.Value.Tunnel.Up, " ")
-	command = command[:len(command)-1]
-	command = append(command, "-F")
-	command = append(command, fmt.Sprintf("%s/ssh.config", viper.GetString("ENV_DIR")))
-
-	pterm.Success.Printfln("Getting tunnel up command")
-
-	err = CallProcess(command[0], command[1:])
-	if err != nil {
-		pterm.Error.Printfln("Running tunnel up command")
-		fmt.Println(err)
-	}
-
-	pterm.Success.Printfln("Running tunnel up command")
-
-	return nil
-}
-
-func (c *tunnelCmd) BastionSHHTunnelStatus() error {
-	sess, err := utils.GetSession(&utils.SessionConfig{
-		Region:  c.config.AwsRegion,
-		Profile: c.config.AwsProfile,
-	})
+	re, err := regexp.Compile(`LocalForward\s(?P<localPort>\d+)\s(?P<remoteHost>.+):(?P<remotePort>\d+)`)
 	if err != nil {
-		pterm.Error.Printfln("Getting AWS session")
+		pterm.Error.Printfln("Getting SSH forward config")
 		return err
 	}
 
-	pterm.Success.Printfln("Getting AWS session")
-
-	ssmSvc := ssm.New(sess)
+	hosts := re.FindAllStringSubmatch(
+		strings.Join(config.SSHForwardConfig.Value, "\n"),
+		-1,
+	)
 
-	out, err := ssmSvc.GetParameter(&ssm.GetParameterInput{
-		Name:           aws.String(fmt.Sprintf("/%s/terraform-output", "dev")),
-		WithDecryption: aws.Bool(true),
-	})
-	if err != nil {
-		pterm.Error.Printfln("Getting tunnel status command")
+	if len(hosts) == 0 {
+		pterm.Error.Printfln("Getting SSH forward config")
 		return err
 	}
 
-	var value []byte
+	pterm.Success.Printfln("Getting SSH forward config")
 
-	value, err = base64.StdEncoding.DecodeString(*out.Parameter.Value)
-	if err != nil {
-		pterm.Error.Printfln("Getting tunnel status command")
-		return err
+	input := &ssm.StartSessionInput{
+		DocumentName: aws.String("AWS-StartPortForwardingSession"),
+		Parameters: map[string][]*string{
+			"portNumber":      {aws.String(strconv.Itoa(22))},
+			"localPortNumber": {aws.String("30022")},
+		},
+		Target: aws.String("i-03a1e68e16db39dea"),
```

Comment:
It sounds like that this value should come from the SSM, let's read from `/<env>/terraform-output`?
Any reason why we changed that?

In the future, I think, we can combine viper and ssm and read/write those params https://paddle.com/blog/go-aws-ssm/